lib/std/os/bits/wasi.zig

```diff
@@ -437,3 +464,23 @@ pub const whence_t = u8;
 pub const WHENCE_SET: whence_t = 0;
 pub const WHENCE_CUR: whence_t = 1;
 pub const WHENCE_END: whence_t = 2;
+
+pub const S_IEXEC = S_IXUSR;
+pub const S_IFBLK = 0x6000;
+pub const S_IFCHR = 0x2000;
+pub const S_IFDIR = 0x4000;
+pub const S_IFIFO = 0xc000;
+pub const S_IFLNK = 0xa000;
+pub const S_IFMT = S_IFBLK | S_IFCHR | S_IFDIR | S_IFIFO | S_IFLNK | S_IFREG | S_IFSOCK;
+pub const S_IFREG = 0x8000;
+// There's no concept of UNIX domain socket but we define this value here in order to line with other OSes.
+pub const S_IFSOCK = 0x1;
+
+pub const SEEK_SET = 0x0; // = __WASI_WHENCE_SET
+pub const SEEK_CUR = 0x1; // = __WASI_WHENCE_CUR
+pub const SEEK_END = 0x2; // = __WASI_WHENCE_END
```

Comment:
Would it be possible to reuse already defined values here rather than setting them manually (as ints) and adding a comment? Something like:

```zig
pub const SEEK_SET = WHENCE_SET;
pub const SEEK_CUR = WHENCE_CUR;
pub const SEEK_END = WHENCE_END;
```

Reply:
done: https://github.com/ziglang/zig/pull/9227/commits/7814a2bd4a3ec22cd9548c622f7dc837dba968f7